crates/orderbook/openapi.yml

```diff
@@ -483,6 +483,30 @@ paths:
           description: Too many order quotes
         500:
           description: Unexpected error quoting an order
+  /api/v1/solver_competition/{auction_id}:
+    get:
+      summary: Information about solver competition
+      description: |
+        Returns the settlements submitted by every solver for a specific auction
+        id. The auction id corresponds to the id external solvers are provided
+        with. Auction ids are not globally unique, they are reused from time to
+        time. The backend keeps the competition information for a limited amount
+        of time.
```

Comment:
Comment by Filius on Discord:

From what I read, auctions (and their solutions) are only stored temporarily on the backend. As a solver developer, the process I would follow to benchmark my solution would be: (i) select an auction ID I would like to benchmark from my own logs, (ii) query solver competition endpoint to obtain maps of solvers -> settlements and (iii) try to understand why the solution with the highest objective value is better than mine. For (iii) I would need to be sure the response I obtained from the endpoint really corresponds to the auction I selected (input -> output). Since auction IDs are not unique, an auction timestamp would help me ensure I am looking at the right auction.

Reply:
We want to eventually store this information in our database for a long time but it might take a bit longer until we have this implemented.  Until then we are planning on using a simple in memory store with some size limit. The auction ids are probably going to be truly unique in the future too but not yet.

Until we have both of that maybe it would reasonable for solver developers to persist the competition information on their own? In their solver implementations they get the auction id and could query the competition info ~a minute after solving and persist it locally.

Reply:
Agree.